solutions/functions/college.js

```diff
@@ -0,0 +1,32 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+function sumofdivisors(n){
+    //Code here
+
+    let requiredSum = 0;
+
+    for(let i = 1; i <= (parseInt(Math.sqrt(n))+1); i++){
```

Comment:
1. Please use `Math.floor`
2. Move `sqrt(n) + 1` outside the `for` loop

Reply:
Addressed.